appengine/firebase-tictactoe/README.md

```diff
@@ -0,0 +1,54 @@
+# Tic Tac Toe on Google App Engine Standard using Firebase
+
+This directory contains a project that implements a realtime two-player game of
+Tic Tac Toe on [Google App Engine] Standard, using the [Firebase] database
```

Comment:
I'd put Standard inside the link and link to https://cloud.google.com/appengine/docs/about-the-standard-environment instead